packages/babel-runtime/scripts/build-dist.js

```diff
@@ -3,6 +3,7 @@ var each       = require("lodash/each");
 var fs         = require("fs");
 var _          = require("lodash");
 
+var join       = require("path").join;
```

Comment:
Unless the changes to this file are critical, I'd be tempted to remove them from this PR to simplify it